examples/dhf-flow/plugins/entities/MDM/harmonize/SmartMaster/SmartMaster.properties

```diff
@@ -1,6 +0,0 @@
-mainModule=main.xqy
```

Comment:
Removed SmartMaster harmonization flow as the gradle task using the sm-match-and-merge REST endpoint is the supported method.